pkg/network/apis/network/v1/doc.go

```diff
@@ -1,7 +1,7 @@
-// +k8s:deepcopy-gen=package,register
 // +k8s:conversion-gen=github.com/openshift/origin/pkg/network/apis/network
+// +k8s:conversion-gen-external-types=github.com/openshift/api/network/v1
 // +k8s:defaulter-gen=TypeMeta
-// +k8s:openapi-gen=true
+// +k8s:defaulter-gen-input=../../../../../../../../github.com/openshift/api/network/v1
```

Comment:
this is pretty ugly... can we improve the generator?

Reply:
> this is pretty ugly... can we improve the generator?

Yeah.  This one in particular is off.  Once we break the back of the problem we can fine tune.